### PR TITLE
Fix some world gen mappings and map some unmapped stuff

### PR DESCRIPTION
**Pull request by valoeghese**

### CODE REVIEW

**Review by liach — APPROVED** [review]

**Review by jaskarth — APPROVED** [review]

File: mappings/net/minecraft/world/biome/Biome.mapping [hunk 1]
```diff
@@ -112,6 +126,8 @@ CLASS net/minecraft/class_1959 net/minecraft/world/biome/Biome
 		METHOD method_8743 scale (F)Lnet/minecraft/class_1959$class_1960;
 			ARG 1 scale
 		METHOD method_8745 parent (Ljava/lang/String;)Lnet/minecraft/class_1959$class_1960;
+			COMMENT Sets the biome that this will replace as a modified version of the biome.
+			COMMENT @param parent the string identifier of the biome to be replaced
```
Comment (liach) [inline mappings/net/minecraft/world/biome/Biome.mapping]: Please comment on the parameter instead of adding a param tag on the method.

**Review by liach — CHANGES_REQUESTED** [review]

File: mappings/net/minecraft/world/biome/Biome.mapping (hunk 1 above)
Comment (valoeghese) [reply mappings/net/minecraft/world/biome/Biome.mapping]: wdym?

Comment (valoeghese) [reply mappings/net/minecraft/world/biome/Biome.mapping]: like this? (see new change)

Comment (valoeghese) [reply mappings/net/minecraft/world/biome/Biome.mapping]: oh wait

Comment (valoeghese) [reply mappings/net/minecraft/world/biome/Biome.mapping]: yeah I see what you mean lol

Comment (valoeghese) [reply mappings/net/minecraft/world/biome/Biome.mapping]: done

**Review by liach — APPROVED** [review]

**Review by NeusFear — APPROVED** [review]

**Review by Runemoro — APPROVED** [review]

**Comment by liach** [comment]
Hope modmuss can get this in before 1.15 release so that we celebrate 1.15 with the first mappings with javadoc in yarn

**Pull request merged**